apis/goharbor.io/v1alpha3/harborcluster_types.go

```diff
@@ -95,12 +95,19 @@ type Database struct {
 type PostgresSQLSpec struct {
 	harbormetav1.ImageSpec `json:",inline"`
 
-	Storage          string                      `json:"storage,omitempty"`
-	Replicas         int                         `json:"replicas,omitempty"`
-	StorageClassName string                      `json:"storageClassName,omitempty"`
-	Resources        corev1.ResourceRequirements `json:"resources,omitempty"`
-	SslConfig        string                      `json:"sslConfig,omitempty"`
-	ConnectTimeout   int                         `json:"connectTimeout,omitempty"`
+	// Storage is define database data store pvc size
+	// +kubebuilder:validation:Optional
+	Storage string `json:"storage,omitempty"`
+	// Replicas is define database instance replicas
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:Minimum:=1
+	Replicas int `json:"replicas,omitempty"`
+	// StorageClassName is define use which StorageClass to create pvc
+	// +kubebuilder:validation:Optional
+	StorageClassName string `json:"storageClassName,omitempty"`
+	// Resources is define database pod resource config
+	// +kubebuilder:validation:Optional
+	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
```

Comment:
As all the fields are optional, do the default values are correctly handled?

Reply:
yes, postgress-operator will set the default value.